node/tests/hubController.js

```diff
@@ -122,7 +125,11 @@ test.serial('translatePlatformsInvalidHmac', async t => {
 
     var verificationInfo = {};
     verificationInfo.key = config.subscriptionInfo.key;
-    verifcationInfo.hmac = "this_wont_match_the_hash";
+    verificationInfo.header = {
+        "X-Hub-Signature": "this_wont_match_the_hash"
+    };
 
-    t.throws(hubController.translatePlatforms(config.hubId, authInfo, config.subscription.sampleFeed, verificationInfo));
```

Comment:
this test will currently fail because the error thrown inside of getPlatforms is caught in the hubController.translatePlatforms function (where it is logged and ignored).  ava t.throws(..) on picks up uncaught errors, so unless the hubController returns a useful error, I've got nothing to work with (except for an undefined value).

---
In reply to: [100191664](https://github.com/openT2T/cloud/pull/14#discussion_r100191664) [](ancestors = 100191664)